Monika After Story/game/script-topics.rpy

```diff
@@ -17740,3 +17740,42 @@ label monika_hedonism:
     m 7etd "It's no wonder most people don't follow that belief...{w=0.3}it's too simple, where morality is complicated."
     m 1eud "So it makes sense why Oscar Wilde portrayed hedonism in a bad light."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_influencers", # TODO - or instagram models?
+            category=['socialmedia? TODO'],
+            prompt="Influencers",
+            random=True,
+        )
+    )
+
+# TODO:
+# the follow any models Q should be pm - but which?
+
+label monika_influencers:
+    # TODO: sprite codes
+    m 1eua "Do you follow any models on social media?{fast}{nw}"
+    # TODO: consider adding quanitifable answers - do they follow a lot (like a lifestyle)
+    #   or do they follow a few (like a like/want)?
+    menu:
+        m "Do you follow any models on social media?"
+        "Yes.":
+            pass
+        "No.":
+            pass
+
+    m 7eud "I find the culture around online 'models' so interesting."
+    m "Before the age of social media people would only see these types of models and personalities on television or in magazines."
+    m "Models were hired by big companies and were told exactly what they would wear and do, they never had a say."
+    m 6eua "Nowadays Instagram models are completely in charge of their outfits and what they chose to do!"
+    m "It seems a lot more ethical now if you ask me. Even with the online sex work industry, these models are completely in charge of what type of NSFW content they’re making and how much money they’re making. "
+    m 2ekc "While it’s mentally it’s probably better for these online celebrities, I think it does have a negative impact on the regular everyday person."
+    m "Just like in the olden days, models are still heavily editing their photos making themselves look unrealistic."
+    m "I think this could be damaging to young girls or even young men who think all women should look like these altered versions of women."
+    m "I have often thought down on myself because of this,"
+    show monika 5eubfa
+    extend "but thankfully you like me just the way I am hehe~"
+    return
```

Comment:
init 5 python:
    addEvent(
        Event(
            persistent.event_database,
            eventlabel="monika_influencers", # TODO - or instagram models?
            category=['socialmedia? TODO'],
            prompt="Influencers",
            random=True,
        )
    )

# TODO:
# the follow any models Q should be pm - but which?

label monika_influencers:
    m 1eua "Do you follow any models on social media?{fast}{nw}"
    # TODO: consider adding quanitifable answers - do they follow a lot (like a lifestyle)
    #   or do they follow a few (like a like/want)?
    menu:
        m "Do you follow any models on social media?"
        "Yes.":
            pass
        "No.":
            pass

    m 7eud "I find the culture around online 'models' so interesting."
    m 7euc "Before the age of social media people would only see these types of models and personalities on television or in magazines."
    m 2rud "Models were hired by big companies and were told exactly what they would wear and do, they never had a say."
    m 2eua "Nowadays Instagram models are completely in charge of their outfits and what they chose to do!"
    m 4eub "It seems a lot more ethical now if you ask me. Even with the online sex work industry, these models are completely in charge of what type of NSFW content they’re making and how much money they’re making. "
    m 2ekc "While it’s mentally it’s probably better for these online celebrities, I think it does have a negative impact on the regular everyday person."
    m 2ekd "Just like in the olden days, models are still heavily editing their photos making themselves look unrealistic."
    m 3gkd "I think this could be damaging to young girls or even young men who think all women should look like these altered versions of women."
    m 1dkd "I have often thought down on myself because of this, {w=0.2}{nw}"
    show monika 5eubfa at t11 zorder MAS_MONIKA_Z with dissolve_monika
    extend 5eubfa "but thankfully you like me just the way I am hehe~"
    return

Reply:
@my-otter-self you want to review such things per-line (or a few lines if needed), not the whole thing at once. Otherwise it's hard to line up existing code/text/exp with your suggestions.

Reply:
noted. want me to delete and do it again?

Reply:
Up to you